### ISSUE TITLE
adding skeleton build instruction

### ISSUE THREAD

**Issue by HinTak** [HinTak/Font-Validator#29]
Moving the build instruction section to wiki does have a slight problem that a git clone does not come with build instruction anymore. In any case, the wiki have other misc build-related info, and probably shouldn't be in the code. Need re-adding brief build instruction plus pointer to wiki for more info

**Comment by Pomax** [comment]
hm, kind of sounds like what a `./docs` dir is useful for, rather than a wiki (and if the docs are in .md form, github will even automatically render them as webpages through github.io, literally no additional work required)

**Comment by HinTak** [comment]
Yes, a `docs` directory is probably how things should go. The stuff that was moved to the wiki isn't quite how it should be, in any case; and really only covers building the core C# part of things. Missing are:

- building the CHM (requires wine and Microsoft tools)

- building the freetype backends . The windows backend is fairly vanilla mingw, but the mac OS X one requires a custom cross compiler, the ubuntu linux backends uses docker .

- the packaging instructions for the rather wonderful FontVal 2.1.1 binaries . Only mono 4.8 is known to work; mono 4.6.0 is too old; mono 5.2.0 is known to be broken; no idea about mono 5.0 or the upcoming 5.4. 

- a fair amount of de-blob-ing for a number of bundled things and build them from source like SharpFont, etc.
